### PR TITLE
app-misc/ddccontrol: add 0.6.0

### PR DESCRIPTION
**Pull request by reagentoo**
```
app-misc/ddccontrol: add 0.6.0

Closes: https://bugs.gentoo.org/829719
Signed-off-by: Dmitry Baranov <reagentoo@gmail.com>

app-misc/ddccontrol-db: add 20210812

Closes: https://bugs.gentoo.org/829720
Signed-off-by: Dmitry Baranov <reagentoo@gmail.com>
```

### CODE REVIEW

**Review by juippis — APPROVED** [review]

**Comment by reagentoo** [comment]
@juippis do we need to bump EAPI?

**Comment by juippis** [comment]
I already have, will fix-on-merge. Thanks for noticing though!

File: app-misc/ddccontrol-db/ddccontrol-db-20210812.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+SRC_URI="https://github.com/ddccontrol/ddccontrol-db/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="nls"
+
+RDEPEND="nls? ( sys-devel/gettext )"
+DEPEND="${RDEPEND}
+		dev-util/intltool
```
Comment (thesamesam) [inline app-misc/ddccontrol-db/ddccontrol-db-20210812.ebuild:19]: BDEPEND, usually.

File: app-misc/ddccontrol/ddccontrol-0.6.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	dev-libs/libxml2:2
+	app-arch/xz-utils
+	gtk? ( x11-libs/gtk+:2 )
+	nls? ( sys-devel/gettext )
+	pci? ( sys-apps/pciutils )"
+DEPEND="${RDEPEND}
+	dev-perl/XML-Parser
+	dev-util/intltool
+	sys-kernel/linux-headers
+	doc? (
```
Comment (thesamesam) [inline app-misc/ddccontrol/ddccontrol-0.6.0.ebuild:27]: At least intltool (probably) and the doc? bit are BDEPEND

File: app-misc/ddccontrol/ddccontrol-0.6.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+			-e '/i810_/d' \
+			src/ddcpci/main.c || die
+	fi
+
+	eapply_user
+
+	## Save for a rainy day or future patching
+	touch config.rpath ABOUT-NLS
+	eautoreconf
+	intltoolize --force || die "intltoolize failed"
```
Comment (thesamesam) [inline app-misc/ddccontrol/ddccontrol-0.6.0.ebuild:58]: Uh..

**Comment by juippis** [comment]
e4d323ee2417623b3fd59d903b332c3de2cc10af
94a047564dc587422a74b92174038552fdab732e
Here are the "fix" commits - if I missed anything, let me know @thesamesam !

**Pull request closed without merging**